package.json

```diff
@@ -11,37 +11,41 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.0.0",
-    "@angular/common": "~8.0.0",
-    "@angular/compiler": "~8.0.0",
-    "@angular/core": "~8.0.0",
-    "@angular/forms": "~8.0.0",
-    "@angular/platform-browser": "~8.0.0",
-    "@angular/platform-browser-dynamic": "~8.0.0",
-    "@angular/router": "~8.0.0",
-    "rxjs": "~6.4.0",
-    "tslib": "^1.9.0",
+    "@angular/animations": "~8.1.1",
+    "@angular/cdk": "^8.1.1",
```

Comment:
Is there any particular reason we have ^ vs ~ for any of these dependencies, or are we just relying on whatever install gives us?

Reply:
Just relying on install, but seems good to have the carrot to match the most recent minor version for the specified major version?

Reply:
Either way. Our own project relies on ^ for minors and below (my recommendation as it is more relaxed). Tilde rarely exists for us; up to you if you want to keep it stricter for patches only. Just be cognisant of what you've used so you remember what to publish to affect production.